<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PI54DLS" failures="1" tests="4" name="Test" time="0.091" errors="0" timestamp="2024-08-03T10:44:20 IST">
  <testcase classname="testng.RetryTest" name="demo" time="0.019">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.003">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.001">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng.RetryTest.demo(RetryTest.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
</testsuite> <!-- Test -->
